src/main/java/seedu/bankwithus/user/AccountList.java

```diff
@@ -346,9 +346,7 @@ public Boolean foundAccountToDelete(String name, Account acc) {
         if (acc.getAccountName().equals(name)) {
             accounts.remove(acc);
             ui.showAccountDeleted(name);
```

Comment:
should add the `doesNameExist.remove(name);` right below the `ui.showAccountDeleted(name);` .

No need to remove the following code as we cannot allow users to delete all remaining accounts without having one account intact:
```
if (accounts.size() < 1) {
//would be good instead if you could help to add a ui message here  to tell users that 
//they need to create a new account because there isn't any active account left.
//once done can close issue #120 when u merge this PR
       createNewAccount();
}
```